devicon.json

```
@@ -3146,6 +3146,36 @@
         "color": "#008cba",
         "aliases": []
     },
+    {
+        "name": "framermotion",
+        "altnames": [
+            "framer-motion",
+            "framer"            
+        ],
+        "tags": [
+            "library",
+            "open-source",
+            "react",
+            "animation"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original",
+                "original-wordmark"
+            ]
+        },
+        "color": "#000",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "original-wordmark"
+            }
+        ]
```

The idea behind aliases works like:
- By default an icon is an `original` version.
- Here is a flowchart about the possible bases and aliases:
```mermaid
flowchart LR;
original --> original-wordmark --> plain-wordmark
original-wordmark --> line-wordmark
original --> plain
original --> line
```
- Remember that the `base` is always an existent icon.
- In this case we have `original` and `original-wordmark`, both working as plain icons, solid colors and one path. Therefore, we can from the `base: original` make an `alias: plain`, same goes to `base: original-wordmark` to `alias: plain-wordmark`.
- Here is an example of complex aliases: #1465

```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            },
            {
                "base": "original-wordmark",
                "alias": "plain-wordmark"
            }
        ]
```

---

Nice chart! 🚀 We should probably save that in the Wiki.